test/e2e/data/infrastructure-vsphere-govmomi/main/topology-runtimesdk/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```

Comment:
Do you remember how it works that this cluster template will use the correct ClusterClass when running the test?

I see how this can work for "quick-start" and "quick-start-supervisor". I have no clue how it would work for the RuntimeSDK ClusterClass

Reply:
Oh wow, this is a huge finding. I thought it works because the runtime sdk tests went through (and by that, it was able to use the config and did because of that reach out to the hooks). But it did not use the correct clusterclass 🤦

Reply:
That's what I was worried about, but wasn't sure :D

Reply:
Resolving here already as the changes address this finding. Now just have to get it back to green after that obviously :D